wemix/bind/structs.go

```diff
@@ -531,18 +556,18 @@ var DefaultInitEnvStorage InitEnvStorage = InitEnvStorage{
 	BALLOT_DURATION_MIN:                      big.NewInt(86400),
 	BALLOT_DURATION_MAX:                      big.NewInt(604800),
 	STAKING_MIN:                              new(big.Int).Mul(big.NewInt(1500000), big.NewInt(params.Ether)),
-	STAKING_MAX:                              new(big.Int).Mul(big.NewInt(1500000), big.NewInt(params.Ether)),
+	STAKING_MAX:                              new(big.Int).Sub(new(big.Int).Lsh(common.Big1, 256), common.Big1), // type(uint256).max
 	MAX_IDLE_BLOCK_INTERVAL:                  big.NewInt(5),
 	BLOCK_CREATION_TIME:                      big.NewInt(1000),
 	BLOCK_REWARD_AMOUNT:                      new(big.Int).Mul(big.NewInt(1), big.NewInt(params.Ether)),
 	MAX_PRIORITY_FEE_PER_GAS:                 new(big.Int).Mul(big.NewInt(100), big.NewInt(params.GWei)),
-	BLOCK_REWARD_DISTRIBUTION_BLOCK_PRODUCER: big.NewInt(4000),
-	BLOCK_REWARD_DISTRIBUTION_STAKING_REWARD: big.NewInt(1000),
+	BLOCK_REWARD_DISTRIBUTION_BLOCK_PRODUCER: big.NewInt(5000),
+	BLOCK_REWARD_DISTRIBUTION_STAKING_REWARD: big.NewInt(0),
```

Comment:
When I gave this value as 0, there was an error when initializing gov contact.

Reply:
It was solved because I was told that if we give it 0, it will be excluded from the initialize parameter.